### PR TITLE
Read queue details from env variable using queue link

### PR DESCRIPTION
**Pull request by patilgirish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,22 +18,28 @@ export enum EventDirection {
...
-| variable       | description                              |
-| -------------- | ---------------------------------------- |
-| LOG_LEVEL      | log level for pino, defaults to debug    |
-| AWS_REGION     | AWS region, defaults to ap-southeast-2   |
-| APPDATA_BUCKET | S3 bucket for storing backed-up messages |
-| APPDATA_PREFIX | S3 prefix for storing backed-up messages |
+| variable          | description                              |
+| ----------------- | ---------------------------------------- |
+| LOG_LEVEL         | log level for pino, defaults to debug    |
+| AWS_REGION        | AWS region, defaults to ap-southeast-2   |
```
Comment (ml019) [inline README.md:37]: Still needed? maybe optional is url used

File: README.md [hunk 2]
```diff
@@ -17,22 +18,28 @@ export enum EventDirection {
 
 ```
 
-**queueUrl**(required) SQS queue url
+**queueLink**(required) Prefix to get the queue details from environment variable, for example
```
Comment (ml019) [inline README.md]: If we make this mandatory, then we drop the url. I was thinking they would be alternatives. If dropping it, then we shouldn't need AWS_REGION either

**Review by ml019 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (patilgirish) [reply README.md:37]: still need it for s3, otherwise uses Us regions as default

File: README.md (hunk 2 above)
Comment (patilgirish) [reply README.md]: right, I will update the readme

**Review by ml019 — APPROVED** [review]
Reflects discussions

**Pull request merged**